index.js

```diff
@@ -118,5 +118,10 @@ bookStore.onUpdate(() => {
 });
 
 window.addEventListener('load', () => {
+  const today = new Date();
+  const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
+  const time = `${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
+  const dateTime = `${date} ${time}`;
+  document.getElementById('currentDateTime').innerText = dateTime;
```

Comment:
- [ ] The current date is displayed below the navigation bar, for this requirement, students should use the  [Luxon](https://moment.github.io/luxon/api-docs/index.html) library. See Get the current date and time
=> This requirement is not fully implemented, the date and time is present but you have to use [Luxon](https://moment.github.io/luxon/api-docs/index.html) as it was required in project requirement, try that out so that it will help you learn more about different libararies

Reply:
Thank you, we are now using the Luxon library to display the date